packages/Avalonia/AvaloniaRules.Project.xml

```diff
@@ -0,0 +1,81 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Rule Name="Avalonia" DisplayName="Avalonia" Order="70" PageTemplate="generic" xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.Categories>
+    <Category Name="Compiling" DisplayName="Compiling" />
+    <Category Name="Debug" DisplayName="Debugging" />
+    <Category Name="NameGenerator" DisplayName="Nameming Generator" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile" HasConfigurationCondition="false" Label="Avalonia" />
+  </Rule.DataSource>
+
+  <!-- Compiling -->
+  <BoolProperty Name="AvaloniaUseExternalMSBuild"
+                DisplayName="External MSBuild"
+                Description="Use dotnet build instead of MSBuild"
+                Category="Compiling" />
```

Comment:
And this is used only internally for the in-solution builds of avalonia source code. Is there any benefit for the non-contributors?

Reply:
> Use dotnet build instead of MSBuild

This is confusing as `dotnet build` is also using MSBuild, while the MSBuild which `dotnet build` is using is not running on .NET Framework.

Reply:
Do you suggest removing it? I've used this in a few cases when I was having trouble debugging.

Reply:
Yes. And this property still can be used from the csproj. It's just not needed for the huge majority of users and is mostly confusing for them.